3.8.0/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM tomcat:8.5-jre8
```

Comment:
Please use `tomcat:8.5-jdk8` instead. 
See https://github.com/docker-library/tomcat/pull/158 to check the reasons.

Reply:
is Ready